### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.10 es 4.71 (13 novembre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#323] [Esercizi M-Z]
![immagine](https://github.com/user-attachments/assets/adf471e6-d9cf-4594-9f6b-c7e8f4d868d0)

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/f1f9a3c4-081d-47dc-b244-1598eebb56eb)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/e1b82857-610c-4f04-a393-71ba80349b82)
